.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
.gitignore file needs to created before pushing your code!

Reply:
I did not know about .gitignore files so I would delete server and client/side node modules when pushing